File: practice/programiz/display_characters_from_a_to_z.c
#include <stdio.h>
/**
 * @brief this function display the characters from a to z
 */
void character_display()
{
    for (int character = 'A'; character <= 'Z'; character++)
    {
        printf("%c ", character);
    }
}

int main(void)
{
    character_display();
    return 0;
}

File: practice/python/display_characters_from_a_to_z.py
import string

# this function display the characters on a std 
def display_character():
    for character in string.ascii_uppercase:
        print(" ", character)

# is function dusari tarabi likh sakate hai
def display_character1():
    for character in range(ord('A'), ord('Z') + 1):
        print(" ", chr(character))
display_character()
display_character1()
print(type(string.ascii_uppercase))